src/main/kotlin/no/nav/familie/ef/sak/fagsak/FagsakPersonController.kt

```diff
@@ -48,8 +56,22 @@ class FagsakPersonController(
                 person.id,
                 overgangsstønad = fagsaker.overgangsstønad?.let { fagsakService.fagsakTilDto(it) },
                 barnetilsyn = fagsaker.barnetilsyn?.let { fagsakService.fagsakTilDto(it) },
-                skolepenger = fagsaker.skolepenger?.let { fagsakService.fagsakTilDto(it) }
-            )
+                skolepenger = fagsaker.skolepenger?.let { fagsakService.fagsakTilDto(it) },
+            ),
+        )
+    }
+
+    @PostMapping("", "/person")
+    fun hentFagsakPersonIdForPerson(@RequestBody personIdentRequest: PersonIdentDto): Ressurs<UUID> {
+        validerIdent(personIdentRequest.personIdent)
+        val personIdenter = personService.hentPersonIdenter(personIdentRequest.personIdent)
+        tilgangService.validerTilgangTilPersonMedBarn(personIdentRequest.personIdent, AuditLoggerEvent.ACCESS)
+        val fagsakPersonId = fagsakPersonService.hentEllerOpprettPerson(
```

Comment:
Burde vi egentlige sjekke om vi har en relasjon til dette personen på en eller annen måte? 
Hvis man faktiskt ønsker er det ju bare å kalle denne endepunktet for å sen få hentet personopplysninger til personen 😬

Reply:
True that. Du tenker vi burde sende inn opprinnelig fagsakPersonId inn her - og slå opp denne i pdl for å dobbeltsjekke at relasjonen er "reell" ?

Reply:
Ja det er en mulighet. Samtidig så har vi vel en annen endepunkt det er allerede er mulig å opprette en fagsak på en ukjent person så har ikke så mye å si.. :P

Reply:
Fin diskusjon her: 
https://nav-it.slack.com/archives/C031MTG3UDP/p1678346360881399
(TL;DR )
Tror vi kom fram til følgende: vi implementerer uten validering og beskriver problematikk i ROS dokument. Vi anser risiko for liten.